src/main/java/com/swyp3/babpool/infra/auth/AuthJwtParser.java

```diff
@@ -35,11 +37,11 @@ public Claims parsePublicKeyAndGetClaims(String idToken, PublicKey publicKey) {
                     .parseClaimsJws(idToken)
```

Comment:
C : `parser()` 와 `setSigningKey()` 메서드는 deprecated 된 메서드 입니다. 아래와 같이 코드를 수정해보면 어떨까요?
```suggestion
return Jwts.parserBuilder()
                 .setSigningKey(publicKey)
                 .build()
                 .parseClaimsJws(idToken)
                 .getBody();
```

Reply:
반영해서 새로운 pr 올렸습니다~